js-weather/index.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html lang="ja">
+  <head>
+    <meta charset="uft-8">
+    <title>Weather</title>
+    <link rel="stylesheet" href="style.css">
+  </head>
+  <body>
+    <div class="container">
+    <h1>現在の天気</h1>
+    <div id="weather"></div>
+    <img id="icon" src="">
+    <form id="city">
+      <select id="selectcity">
+        <option value="london">ロンドン</option>
+        <option value="tokyo">東京</option>
+        <option value="paris">パリ</option>
+        <option value="new york">ニューヨーク</option>
+        <option value="indonesia">インドネシア</option>
+      </select>
+    </form>
+    <div id="cityweather">
+      <div class="result">
+        気温：
+        <div id="temp"></div>
+      </div>
+      <div class="lesutl">
+        湿度：
+        <div id="humidity"></div>
+      </div>
+    </div>
+  </div>
+<script src="index.js" class="icon"></script>
```

Comment:
```suggestion
<script src="index.js"></script>
```
属性 class="icon" は要らなそうなので消しましょう